mlflow/tensorflow.py

```diff
@@ -497,15 +499,16 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         pass
 
+    def on_train_begin(self, logs=None):  # pylint: disable=unused-argument
+        config = self.model.optimizer.get_config()
+        for attribute in config:
+            try_mlflow_log(mlflow.log_param, "opt_" + attribute, config[attribute])
+
```

Comment:
Just adding a comment so you don't forget to move the summary logging here as well!

Reply:
Done, thank you! :D